changelog.txt

```diff
@@ -1,3 +1,8 @@
+4.7.0
+============
+New Features:
+Added Subject Name + Issuer authentication to the acquire token by authorization code and acquire token by refresh token flows with the WithSendX5C() api on the confidential client application.
+
```

Comment:
nit: can you set it up like the other notes, w/a similar format, we have issue in bold and customer impact after that and then a link to the issue. thx.